portfolio/src/main/java/com/google/sps/authentication/servlets/LoginServlet.java

```diff
@@ -13,8 +14,24 @@
 @WebServlet("/login")
 public final class LoginServlet extends HttpServlet {
 
+    private static final String GUEST_USER_NAME = "stranger";
+    private static final String URL_TO_REDIRECT_TO_AFTER_LOGS_OUT = "/index.html";
+    private static final String URL_TO_REDIRECT_TO_AFTER_LOGS_IN = "/comments.html";
```

Comment:
What if the website has many pages, how would login servlet know which page to navigate to after logging in?

Reply:
hmm.... that is a problem, do u have any suggestion towards it?

Reply:
Maybe you can pass it in the param?

Reply:
Actually, since this is not a proper login service. Just to get the login status. Do this class need to care about these 2 paths? Maybe scope down this service to return login status true/false and user name?

Reply:
As discussed offline, since UserService API needs it in servlet, i will just isolate it in the server